village/src/components/SmurfForm.js

```diff
@@ -1,25 +1,30 @@
-import React, { Component } from 'react';
+import React, { Component } from "react";
+import axios from "axios";
 
 class SmurfForm extends Component {
   constructor(props) {
     super(props);
     this.state = {
-      name: '',
-      age: '',
-      height: ''
+      name: "",
+      age: "",
+      height: ""
     };
+    console.log(this.props);
```

Comment:
Don't forget to clean these up, imagine the users/other devs browsing the page start coming across this stuff, and maybe its sensitive data...